### PR TITLE
Docs text improvements

### PR DESCRIPTION
**Pull request by Yung-Beef**
Updated and created new text to reflect the current status of the project and provide more detail, such as explaining how NRG will work and how content storage works.

Fixed some internal links.

### CODE REVIEW

File: docs/basics/faq/Overview.md [hunk 1]
```diff
@@ -112,29 +104,35 @@ Yes, you can stake tokens that have not vested yet.
...
 ## Where can I see Subsocial's smart contracts?
 
-There are no smart contracts for Subsocial, because it is built as a custom blockchain on top of the [Substrate](https://substrate.dev/) blockchain framework. The Subsocial blockchain is optimized for social networking types of operations, and has very low fees.
+There are no smart contracts for Subsocial, because it is built as a custom blockchain on top of the [Substrate](https://substrate.dev/) blockchain framework. 
+The Subsocial blockchain is optimized for social networking types of operations, and has very low fees.
 
 ## Do you have a lightpaper?
 
-[Subsocial Lightpaper](/docs/basics/)
+[Subsocial Lightpaper](/docs/basics/lightpaper/introduction)
```
Comment (tusharojha) [inline docs/basics/faq/Overview.md:112]: Please reverse this change, the /docs/basics page takes user to introduction page only.

File: sidebars.js [hunk 2]
```diff
@@ -9,9 +9,12 @@ module.exports = {
             Architecture: [
               "basics/lightpaper/architecture/architecture",
               "basics/lightpaper/architecture/posts-and-spaces",
+              "basics/lightpaper/architecture/content-storage",
               "basics/lightpaper/architecture/identity",
+              "lightpaper/architecture/energy",
```
Comment (tusharojha) [inline sidebars.js:14]: this should be `basics/lightpaper/architecture/energy`.

File: docs/basics/tokenomics/Token-economics.md [hunk 3]
```diff
@@ -48,4 +51,4 @@ these features are still in development.
 - Minting social tokens
 - Voting in governance
  
-See the [Lightpaper](/docs/basics/) for greater details.
+See the [Lightpaper](/docs/basics/lightpaper/introduction) for greater details.
```
Comment (tusharojha) [inline docs/basics/tokenomics/Token-economics.md:54]: Reverse this change too.

File: docs/basics/tokenomics/Resources.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ id: resources
 title: Resources
 ---
 
-- ([Subsocial Lightpaper](/docs/basics/))
+- ([Subsocial Lightpaper](/docs/basics/lightpaper/introduction))
```
Comment (tusharojha) [inline docs/basics/tokenomics/Resources.md:6]: Reverse it.

**Review by tusharojha — CHANGES_REQUESTED** [review]
I have added a few comments. 

Additionally, it's good if we prefer adding the sidebar property in headers. There are these 4 types of sidebars:

- homeSidebar
- tutorialsSidebar
- developSidebar
- announcementSidebar

and at the top of any markdown file you can add it as follows:

`displayed_sidebar: developSidebar`

File: docs/basics/faq/Overview.md [hunk 5]
```diff
@@ -124,17 +117,22 @@ There are no smart contracts for Subsocial, because it is built as a custom bloc
...
+Install the **Polkadot.js** extension in your browser. This extension allows you to sign transactions for the Subsocial blockchain. 
+It’s similar to MetaMask (a browser extension for Ethereum).
 
 - On Chrome, install via [Chrome web store](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd)
 - On Firefox, install via [Firefox add-ons](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/)
 
-After the Polkadot extension is installed, create (or import) your account and then you can sign in on the Subsocial web dapp (decentralized application) by visiting this [link](https://app.subsocial.network). To use the dapp, you will need Subsocial tokens.
+After the Polkadot extension is installed, create (or import) your account and then you can sign in on the 
+Subsocial web dapp (decentralized application) by visiting this [link](https://app.subsocial.network). 
+A tutorial on how to install and use Polkadot.js is available [here](/docs/tutorials/getting-started/account-setup/polkadot-js).
```
Comment (tusharojha) [inline docs/basics/faq/Overview.md:128]: Just use `docs/tutorials/`. Please copy the paths from docs.subsocial.network and remove base URL.

File: docs/tutorials/TokenClaim.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ However, if you have the seed phrase for your Ledger account, you can import you
 and then you will be able to claim.
 
 ## Can I claim from my phone?
-Yes, please see the tutorial [here](https://docs.subsocial.network/docs/faq/mobile-claim).
+Yes, please see the tutorial [here](docs/tutorials/mobile-claim).
```
Comment (tusharojha) [inline docs/tutorials/TokenClaim.md:36]: This page does not exist.

**Review by tusharojha — CHANGES_REQUESTED** [review]
Added a few more comments that are causing the build to fail.

File: docs/tutorials/TokenClaim.md (hunk 6 above)
Comment (Yung-Beef) [reply docs/tutorials/TokenClaim.md:36]: image
It does though, I've quadruple checked it

**Review by olehmell — APPROVED** [review]

**Pull request merged**